texttospeech/list_voices/list_voices.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+package main
+
+// Command list_voices lists the available text to speech voices.
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+
+	"golang.org/x/net/context"
+
+	texttospeech "cloud.google.com/go/texttospeech/apiv1"
+	texttospeechpb "google.golang.org/genproto/googleapis/cloud/texttospeech/v1"
+)
+
+// [START tts_list_voices]
+func ListVoices(out io.Writer) error {
+	// ListVoices lists the available text to speech voices.
```

Comment:
Same comment - please move this to the line right above the `func` line. GoDoc grabs the comment right above the function declaration and puts it into the function's documentation. So, we also need a newline after the `[START...]`.

```go
// [START tts_list_voices]

// ListVoices lists the available text to speech voices.
func ListVoices...
```

Reply:
Got it, moved. Thanks